### PR TITLE
Edit http link in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by phurinaix**
Edit http link (Creating a pull request)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,9 +15,9 @@
 
 ## WET v4.0
 
-* [Créer un &#171;&#160;pull request&#160;&#187;](http://wet-boew.github.io/wet-boew/docs/pull-fr.html)
+* [Créer un &#171;&#160;pull request&#160;&#187;](https://wet-boew.github.io/wet-boew/docs/pull-fr.html)
 
 ## WET v3.1
 
 * [Développer pour la BOEW](http://wet-boew.github.io/v3.1-ci/docs/gs-cd/dev-fr.html)
```
Comment (nschonni) [inline CONTRIBUTING.md]: What about this one? May as well do all the links in the document at the same time

**Review by nschonni — CHANGES_REQUESTED** [review]

**Comment by nschonni** [comment]
Thanks for the PR, but could you do the rest if we're going to touch that document? ✨

**Comment by phurinaix** [comment]
Thanks for requested changes sir
I thought that was broken link.

**Comment by nschonni** [comment]
@LaurentGoderre I just noticed that the 3.1 submodules got removed https://github.com/wet-boew/wet-boew.github.io/commit/fa924e5c481742d39a016bcac978100456a390b7#diff-8903239df476d7401cf9e76af0252622
Should that section just come out of the guide then?

**Comment by duboisp** [comment]
WET 3.1 is not supported. I don't have an issue if the reference to WET 3.1 contribution are removed from this PR or from another one. 

/cc @lucas-hay

**Comment by LaurentGoderre** [comment]
Yeah same here

**Comment by duboisp** [comment]
@phurinaix Can you update your PR and remove the reference to WET 3.1, line 22.

**Review by nschonni — APPROVED** [review]

**Pull request merged**